packages/block-library/src/button/deprecated.js

```diff
@@ -780,10 +794,12 @@ const deprecated = [
 		isEligible: ( attributes ) =>
 			!! attributes.customTextColor ||
 			!! attributes.customBackgroundColor ||
-			!! attributes.customGradient,
+			!! attributes.customGradient ||
+			!! attributes.align,
 		migrate: compose(
 			migrateBorderRadius,
-			migrateCustomColorsAndGradients
+			migrateCustomColorsAndGradients,
+			migrateAlign
```

Comment:
Is this for general alignment or `textAlignment` specifically?

Reply:
This is for general align, in this PR, we remove general align and add textAlign. Here we migrating the general alignment.